build.gradle

```diff
@@ -304,21 +304,28 @@ configurations {
 }
 
 dependencies {
-    ktlint "com.github.shyiko:ktlint:0.31.0"
+    ktlint("com.pinterest:ktlint:0.48.2") {
+        attributes {
+            attribute(Bundling.BUNDLING_ATTRIBUTE, getObjects().named(Bundling, Bundling.EXTERNAL))
+        }
+        
+    }
 }
 
-task ktlint(type: JavaExec, group: "verification") {
+tasks.register("ktlint", JavaExec) {
+    group = "verification"
     description = "Check Kotlin code style."
     classpath = configurations.ktlint
-    main = "com.github.shyiko.ktlint.Main"
-    args "${projectDir}/components/**/*.kt", "${projectDir}/gradle-plugin/**/*.kt", "buildSrc/**/*.kt", "!**/build", "!**/templates", "!components/external"
+    main = "com.pinterest.ktlint.Main"
```

Comment:
pinterest 😏

Reply:
Hah yeah that was a interesting one -- but in fenix we trust! https://github.com/mozilla-mobile/firefox-android/blob/18764588cd714e2ab18aaa107308c29c6c6f5620/fenix/build.gradle#L226